### PR TITLE
[DISCO-2348] docs update

### PR DESCRIPTION
**Pull request by taddes**
Simple updates in `README` file for using pyenv virtualenv and some updates to running the server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,21 @@ reducing the need for deployments of the services that use it.
...
+pyenv and virtual environments. Using virtualenv creates a .python-version file  in the project directory and if you add `eval "$(pyenv virtualenv-init -)"` to your  `./bashrc` or `./zshrc`, the environment will automatically be activated.
+
+Ex:
+```shell
+export PATH="$HOME/.pyenv/bin:$PATH"
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+```
+
+You also need to ensure you have Postgres (pgAdmin) installed. You can select a distribution from [https://www.postgresql.org/download/](https://www.postgresql.org/download/), use homebrew or their edb installer client.  Make sure to install version 14.7, or equivalent and have it running when developing for consvc-shepherd.
```
Comment (tiftran) [inline README.md:22]: I would exclude the mention of pgAdmin, considering there are other ways one could manager their db.

File: README.md [hunk 2]
```diff
@@ -16,18 +30,36 @@ and rename it to `env`:
 ```shell
 DEBUG=true
 SECRET_KEY=keyboard-mash
+DB_NAME=postgres
+DB_USER=postgres
+DB_HOST=localhost
+DB_PASS=postgres
```
Comment (tiftran) [inline README.md:36]: Can we add these into .env.example as well?

**Review by tiftran — APPROVED** [review]
Thanks for the update! Just some considerations to make

File: README.md (hunk 1 above)
Comment (taddes) [reply README.md:22]: Great point. I made a change that lists it as an option, but not the de-facto way, including cli or config files. Lmk if you think that's fine!

File: README.md (hunk 2 above)
Comment (taddes) [reply README.md:36]: Yes! Good call

**Pull request merged**